src/functionBased/components/Navbar.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { MdClose } from 'react-icons/md';
+import { FiMenu } from 'react-icons/fi';
+
+const Navbar = () => {
+  const links = [
+    {
+      id: 1,
+      path: '/',
+      text: 'Home',
+    },
+    {
+      id: 2,
+      path: '/about',
+      text: 'About',
+    },
+  ];
+  const [navbarOpen, setnavbarOpen] = useState(false);
+  const handleToggle = () => {
+    setnavbarOpen((prev) => !prev);
+  };
+  const closeMenu = () => {
+    setnavbarOpen(false);
+  };
+  return (
+    <nav className="navBar">
+      <button type="button" onClick={handleToggle}>
+        {navbarOpen ? (
+          <MdClose style={{ color: '#fff', width: '40px', height: '40px' }} />
+        ) : (
+          <FiMenu style={{ color: '#7b7b7b', width: '40px', height: '40px' }} />
+        )}
+      </button>
+      <ul className={`menuNav ${navbarOpen ? 'showMenu' : ''}`}>
+        {links.map((link) => (
+          <li key={link.id}>
+            <NavLink
+              to={link.path}
+              // activeClassName="active-link"
```

Comment:
I think this comments is not required as its already defined in another way below.